doc/man3/SSL_CTX_set0_CA_list.pod

```diff
@@ -17,10 +28,27 @@ SSL_get0_peer_CA_list - get or set CA list
  int SSL_CTX_add1_to_CA_list(SSL_CTX *ctx, const X509 *x);
  int SSL_add1_to_CA_list(SSL *ssl, const X509 *x);
 
+ void SSL_CTX_set_client_CA_list(SSL_CTX *ctx, STACK_OF(X509_NAME) *list);
+ void SSL_set_client_CA_list(SSL *s, STACK_OF(X509_NAME) *list);
+ STACK_OF(X509_NAME) *SSL_get_client_CA_list(const SSL *s);
+ STACK_OF(X509_NAME) *SSL_CTX_get_client_CA_list(const SSL_CTX *ctx);
+ int SSL_CTX_add_client_CA(SSL_CTX *ctx, X509 *cacert);
+ int SSL_add_client_CA(SSL *ssl, X509 *cacert);
+
  const STACK_OF(X509_NAME) *SSL_get0_peer_CA_list(const SSL *s);
 
 =head1 DESCRIPTION
 
+The following functions can be called on either the server or the client side.
+For TLS versions before 1.3 the list of CA names is only sent from the server
```

Comment:
No strong views, would "1.2 and earlier" be more clear than "before 1.3" or not?